generated/python/gapic-google-cloud-error-reporting-v1beta1/README.rst

```diff
@@ -7,7 +7,7 @@ easy-to-use client library for the `Stackdriver Error Reporting API`_ (v1beta1)
 
 .. _`googleapis`: https://github.com/googleapis/googleapis/tree/master/google/devtools/clouderrorreporting/v1beta1
 .. _`google-gax`: https://github.com/googleapis/gax-python
-.. _`Stackdriver Error Reporting API`: https://developers.google.com/apis-explorer/?hl=en_US#p/clouderrorreporting/v1beta1
+.. _`Stackdriver Error Reporting API API`: https://developers.google.com/apis-explorer/?hl=en_US#p/clouderrorreporting/v1beta1
```

Comment:
googleapis/toolkit#1028 (once that is done, I will likely just fix these manually).